src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1217,6 +1217,10 @@ def set_up_existing_container_update(self):
             # Fails if all containers are removed
             while (len(self.new_containerapp["properties"]["template"]["containers"]) > 1):
                 self.new_containerapp["properties"]["template"]["containers"].pop()
+            if not self.get_argument_name():
+                raise RequiredArgumentMissingError("The ContainerApp name is required when --force-single-container-updates is set.")
```

Comment:
Nit: is is actually ` --force-single-container-updates`? Shouldn't this be just the variable name, `force_single_container_updates`?

Reply:
Updated this error message.